lib/js/index.js

```diff
@@ -126,12 +126,21 @@ class DateTimePicker extends Events {
     this.$('.js-ok')
       .addEventListener('click', () => this.clickSubmit(), false);
 
+    this.$('.c-datepicker__header-date__hours').addEventListener('click', e => this.showHourClock(e), false);
+    this.$('.c-datepicker__header-date__minutes').addEventListener('click', e => this.showMinuteClock(e), false);
+
     this.$('.c-datepicker__clock__hours').addEventListener('mouseleave', e => this.mouseOutClock(e), false);
-    this.$(`.${this.options.styles.clockNum}`).forEach((el) => {
+    this.$(`.c-datepicker__clock__hours .${this.options.styles.clockNum}`).forEach((el) => {
       el.addEventListener('click', e => this.clickClock(e), false);
       el.addEventListener('mouseenter', e => this.mouseInClock(e), false);
     });
 
+    this.$('.c-datepicker__clock__minutes').addEventListener('mouseleave', e => this.mouseOutClock(e), false);
```

Comment:
`.js-` class please. I'll stop making this comment now, but it should apply to each new className referenced in JavaScript.

I'm going to do the same to any pre-existing non `js-` classes soon.